### PR TITLE
Dessert

### PR DESCRIPTION
**Pull request by saiisaai**

### CODE REVIEW

**Review by Supakrit65 — CHANGES_REQUESTED** [review]
please remove .vscode file 👍

**Review by premepreme — APPROVED** [review]
ok but pls remove .vscode

File: README.md [hunk 1]
```diff
@@ -30,5 +30,5 @@ Section1 | Section2 | Section3 | ...
 
 | Name      | Role      | Github          |
 |:----------|-----------|-----------------|
-| Your name | Your menu | @your_github_id |
+| Chanakarn | Dessert | [@saiisaai](https://github.com/saiisaai) |
```
Comment (ishiasaka) [inline README.md]: Name is full name plz.

**Review by ishiasaka — CHANGES_REQUESTED** [review]

**Review by teenteennnii — APPROVED** [review]
OK

**Review by premepreme — APPROVED** [review]
ok

**Review by ishiasaka — APPROVED** [review]
Yay. It's good 👍

**Pull request merged**